### DISCUSSION TITLE
Getting String scalar value in Unicode to support language text

### DISCUSSION THREAD

**Discussion by BigBallard** [charleskorn/kaml#432] [Q&A]
I am working on reading some yaml that has localization maps. When reading the content of the values, the results of the text that are in Chinese for instance, end up being garbage. Does anyone have a method of reading yaml content in Unicode so it can support the nationalized Unicode characters?

**Comment by charleskorn** [comment]
One thing to check is that the file's encoding is correctly set, and that you're specifying the correct encoding when reading the file as well. For example, if you're using `Yaml.decodeFromStream`, make sure to pass the correct `Charset`.
